### ISSUE TITLE
Add to docs that `name` is required on input's for hx-include to work

### ISSUE THREAD

**Issue by wimdeblauwe** [bigskysoftware/htmx#2384]
It is obvious in retrospect, but at first, I could not figure out why my `hx-include="#someElement"` was not working. The reason was that my input looked like this:

```
<input type="date" id="someElement">
```
There was no warning in the JavaScript console and there was no hint in the [hx-include](https://htmx.org/attributes/hx-include/) documentation.

Maybe the "Notes" section could have an extra point like this:

> Ensure all `<input>` elements have a `name` attribute as this will be the name that is used for sending the value to the server. 
> Without it, the value of the input will not be included in the request.

**Comment by andryyy** [comment]
I just wonder how you noticed. Because when there’s no name, what were you looking for? :)

Given it would include that element (while this element is never included in FormData by specification w/o name), what would be included?

**Comment by wimdeblauwe** [comment]
I refered using the `id` (So `hx-include="#someElement"). As I stated "It is obvious in retrospect" :-)

**Comment by schungx** [comment]
HTML forms always uses `name` noy `id`... This is just common practice.

Id's are supposed to be unique while `name` may not.

**Comment by andryyy** [comment]
Since users often include elements that may or may not have form data, it would be confusing to have these warnings in the console. That’s just my thought on it. :)

And thanks for the clarification. :)

**Comment by wimdeblauwe** [comment]
Yeah, I would not add a warning in the console. Just a little note in the docs might go a long way.

**Comment by 1cg** [comment]
heya @wimdeblauwe i'm gonna close this out, doubt a docs note would be read by most people and LLMs would be more help :)
